receiver/elasticsearchreceiver/client.go

```diff
@@ -89,7 +89,7 @@ const nodeStatsMetrics = "breaker,indices,process,jvm,thread_pool,transport,http
 // nodeStatsIndexMetrics is a comma separated list of index metrics that will be gathered from NodeStats.
 const nodeStatsIndexMetrics = "store,docs,indexing,get,search,merge,refresh,flush,warmer,query_cache,fielddata,translog"
 
-const indexStatsMetrics = "search"
+const indexStatsMetrics = "_all"
```

Comment:
We should fetch `_all`, because we've covered all datapoints in #16170. If there will be performance issues, maybe these operations should become a config option.

Reply:
Do you have a sense of what kind of performance impact this will have? Perhaps it's worth a quick manual benchmark.

Reply:
Here's the benchmark I used: https://gist.github.com/aboguszewski-sumo/fcea4d5aad744db6ad20838a500ffa05 
I've run it on a local machine, with local elasticsearch cluster consisting of one node running on Docker.
The first few runs showed a `100%` time increase - scraping `_all` had results of `~2_000_000` ns/op, but it then became closer to `~1_500_000` ns/op (seems like a cold cache or something similar), where scraping only `search` took `~1_000_000` ns/op. So, seems like a time increase between `50%` and `100%`.
I also made a second try, but this time I used all functions from the `elasticsearchClient` interface (four in total, without the one that got approved, but hasn't been merged yes). After some initial runs (to discard the results suspected for being a result of cold cache), scraping only `search` metrics took between `4_700_000` ns/op and `5_000_000` ns/op, while scraping all metrics took around `5_500_000` ns/op - so, more like `10%` to `17%` of time increase.
I know that this benchmark is far from being perfect because of running on everything locally, but it gives us a general sight of how does this affect the performance.

Reply:
Thanks for running the benchmark. Sounds not quite ideal, but I don't think our margins here are not razor thin. ~5ms is still a reasonable baseline in my opinion. If we find that we need to optimize, I think we can do so by adding a mechanism to disable or scope index collection.

Reply:
Great, thanks for the reminder.